README.md

```diff
@@ -150,6 +129,83 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
+
+### Getting Remote User IDs
+
+In order to start a streem with a remote user, your app will need to supply that user's `remoteUserId`. There are two mechanisms available in the SDK for retrieving a `remoteUserId`. The first is through getting a list of recently logged-in users. The second is through an invitation. In addition to these two methods, you can maintain your own list of `remoteUserIds` and supply them to your app however you choose. 
+
+#### Recently Logged In Users
+
+By calling the SDK's `getRecentlyIdentifiedUsers` method on the Streem `sharedInstance` you will receive a list of recently logged in users for your company. You can use this method to filter only experts. You then give this method a callback where you select the remote user through some mechanism and start the streem. 
+
+```swift
+    Streem.sharedInstance.getRecentlyIdentifiedUsers(onlyExperts: true) { [weak self] users in
+        guard let self = self else { return }
+
+        // Select the user you wish to streem with
+
+        Streem.sharedInstance.startRemoteStreem(
+            asRole: .LOCAL_CUSTOMER,
+            withRemoteUserId: selectedUser.id) { success in 
+                // handle succes of streem
+        }
+``` 
+
+#### Invitations
+
+If you are utilizing Streem's Pro implementations on the web and/or iOS you will likely have access to invitations. Invitations are communicated via a 9-digit code. This code can be transmitted through SMS, email, or copy and pasted to some other mechanism such as Slack. 
+
+Once you've retrieved this 9-digit code in your app you will follow three steps. 
+
+* Call `login(withSmsInvitationId:)`. This will retrieve the invitation details for the given code. 
+* Call `identifyCustomer(companyCode:inviteCode:name:avatarUrl:completion:)` to identify the customer with the Streem `sharedInstance`. 
+* Call `startRemoteStreem` with the remote user contained in the invitation. The whole flow looks like:
+
+```swift
+    StreemAuth.sharedInstance.login(withSmsInvitationId: invitationCode) { error, loginResponse, details in 
+        let invitation = Invitation(
+	    requesterName: details.name,
+	    displayName: details.user.displayName,
+	    invitationToken: details.token,
+	    idToken: loginResponse.token,
+	    code: code,
+	    remoteId: details.user.uid,
+	    referenceId: details.referenceId,
+	    photoURL: details.user.photoURL,
+	    companyCode: details.company.companyCode,
+	    companyName: details.company.name,
+	    companyLogoURL: details.company.logoUrl)
+
+        Streem.sharedInstance.identifyCustomer(
+            companyCode: invitation.companyCode,
+            inviteCode: invitation.code,
+            name: invitation.requesterName,
+            avatarUrl: invitation.photoURL) { success in 
+            if success {
+                Streem.sharedInstance.startRemoteStreem(
+                    asRole: .LOCAL_CUSTOMER,
+                    withRemoteExternalUSerId: nil,
```

Comment:
Since this value defaults to `nil`, why don't we just leave it out? Since its no longer needed with the new invitation system, I think it might cause some confusion here. If we're trying to support a different method of starting a streem, that should be probably documented elsewhere.

Reply:
sure